docs/root/configuration/http/http_conn_man/headers.rst

```diff
@@ -587,7 +587,55 @@ Supported variable names are:
       <config_http_conn_man_headers_x-forwarded-for>`.
 
 %DOWNSTREAM_REMOTE_ADDRESS_WITHOUT_PORT%
-    Same as **%DOWNSTREAM_REMOTE_ADDRESS%** excluding port if the address is an IP address.
+    Remote address of the downstream connection. If the address is an IP address the output does
+    *not* include port.
+
+    .. note::
+
+      This may not be the physical remote address of the peer if the address has been inferred from
+      :ref:`Proxy Protocol filter <config_listener_filters_proxy_protocol>` or :ref:`x-forwarded-for
+      <config_http_conn_man_headers_x-forwarded-for>`.
+
+%DOWNSTREAM_REMOTE_PORT%
+    Similar to **%DOWNSTREAM_REMOTE_ADDRESS_WITHOUT_PORT%**, but only extracts
```

Comment:
This seems confusing. We are saying that the port formatter is similar to the one that explicitly excludes the port?

Reply:
Again, I copied the existing language from `access_log/usage.rst` on main. Can you please confirm is that is not accurate, esp. since you committed it?